recipes/simdutf/all/conandata.yml

```diff
@@ -10,6 +10,9 @@ patches:
     - patch_file: "patches/2.0.0-0001-fix-cmake.patch"
       patch_description: "remove static build, stop to link static libc++ and enable rpath on macOS"
       patch_type: "conan"
+    - patch_file: "patches/2.0.0-0002-add-workaround-gcc9.patch"
+      patch_description: "apply gcc8 workaround to gcc9"
+      patch_type: "conan"
```

Comment:
Portability maybe a better fit?

Reply:
I think so.
I am going to resolve all compilation error in older gcc.
After that, I will update patch_description and create PR for upstream.